tgui/packages/tgui/interfaces/IndividualStats/IndividualLoadouts.tsx

```diff
@@ -128,69 +128,71 @@ export const IndividualLoadouts = (props) => {
+            mt={'10px'}
+            mb={'14px'}
+            ml={'10px'}
           >
             Equipment options
           </Stack>
-          <Stack vertical>
-            {available_loadouts_data
-              .filter((potentialitem) => potentialitem.job === selectedJob)
-              .filter(
-                (potentialitem) =>
-                  potentialitem.slot === selectedLoadoutItem.item_type.slot,
-              )
-              .map((potentialitem) => (
-                <Stack.Item key={potentialitem.name}>
-                  <Button
-                    width={'180px'}
-                    onClick={() => {
-                      setselectedPossibleItem(potentialitem);
-                      potentialitem.unlocked &&
-                        act('equip_item', {
-                          selected_item: potentialitem.type,
-                          selected_job: potentialitem.job,
-                        });
-                    }}
-                    color={
-                      selectedPossibleItem.name === potentialitem.name
-                        ? 'orange'
-                        : !potentialitem.unlocked ||
-                            potentialitem.quantity === 0
-                          ? 'grey'
-                          : !potentialitem.valid_choice
-                            ? 'red'
-                            : 'blue'
-                    }
-                  >
-                    <Flex align="center">
-                      {!!potentialitem.icon && (
-                        <Flex.Item
-                          mr={1.5}
-                          className={classes([
-                            'campaign_loadout_items18x18',
-                            selectedPossibleItem.name === potentialitem.name
-                              ? potentialitem.icon + '_orange'
-                              : !potentialitem.unlocked ||
-                                  potentialitem.quantity === 0
-                                ? potentialitem.icon + '_grey'
-                                : !potentialitem.valid_choice
-                                  ? potentialitem.icon + '_red'
-                                  : potentialitem.icon + '_blue',
-                          ])}
-                        />
-                      )}
-                      {potentialitem.name}
-                    </Flex>
-                  </Button>
-                </Stack.Item>
-              ))}
-          </Stack>
+          <Section fill scrollable width={'240px'} height={'555px'}>
+            <Flex width={'220px'} wrap>
+              {available_loadouts_data
+                .filter((potentialitem) => potentialitem.job === selectedJob)
+                .filter(
+                  (potentialitem) =>
+                    potentialitem.slot === selectedLoadoutItem.item_type.slot,
+                )
+                .map((potentialitem) => (
+                  <Flex.Item mr={1.5} mb={1} key={potentialitem.name}>
```

Comment:
we use stack not flex, usually why are you using flex here, you're adding padding anyway

Reply:
As mentioned in discord, stack screws up the formatting. Happy to change it if you know how to make it work with stack though.

Reply:
done